shell/browser/electron_browser_main_parts.cc

```diff
@@ -530,6 +535,35 @@ void ElectronBrowserMainParts::PostMainMessageLoopRun() {
     }
   }
 
+  // Shutdown utility process created with Electron API before
+  // stopping Node.js so that exit events can be emitted. We don't let
```

Comment:
Why do we care that exit events are emitted during shutdown? Presumably it's enough that everything just dies?

Reply:
`exit` event is good signal for users to know that child process was properly reaped even on shutdown, so I think consistently providing this signal would be better ? Also, keeping it consistent with `child_process.fork`.

Reply:
consistency with `fork` is good enough for me, but does this actually reap? it looks like we just fire off the event.

Reply:
Yup we are only firing the event here and shutdown of the browser side wrapper, actual termination of the process is performed after this function exits by the content layer via `content::BrowserProcessIOThread::ProcessHostCleanup`